Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs

```diff
@@ -153,97 +156,174 @@ public async Task<GetStatsResponse> GetStatsAsync(GetStatsRequest request, Cance
             };
         }
 
+        private async Task<OpenStreamResult> GetFileStreamAsync (Context context, ContentHash hash)

```

Comment:
nit: formatting - extra space after the method name and before the opening brace.

Reply:
Fixed.